### ISSUE TITLE
[sqlite] In SQLite loader, exception handler references `sqlite3` without importing it

### ISSUE THREAD

**Issue by jsvine** [saulpw/visidata#1125]
**Small description**

When the SQLite loader encounters an error, it tries to raise an exception ... but the exception handler references `sqlite3`, which has not been imported, raising an exception of its own:

https://github.com/saulpw/visidata/blob/666e30329878b74678d24c300e1c50ceb7aa0666/visidata/loaders/sqlite.py#L58

**Expected result**

No extra exception.

**Actual result with screenshot**

Here's the stacktrace:

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "[...]/visidata/lib/python3.8/site-packages/visidata/threads.py", line 206, in _toplevelTryFunc
    t.status = func(*args, **kwargs)
  File "[...]/visidata/lib/python3.8/site-packages/visidata/sheets.py", line 289, in reload
    for r in self.iterload():
  File "[...]/visidata/lib/python3.8/site-packages/visidata/loaders/sqlite.py", line 125, in iterload
    for row in SqliteSheet.iterload(self):
  File "[...]/visidata/lib/python3.8/site-packages/visidata/loaders/sqlite.py", line 58, in iterload
    except sqlite3.OperationalError:
NameError: name 'sqlite3' is not defined
```

**Steps to reproduce with sample data and a .vd**

```sh
echo "a" > x.sqlite; vd x.sqlite
```

**Additional context**

VisiData v2.6 (PyPI release)

**Comment by saulpw** [comment]
Thanks @jsvine!  I've fixed this locally and we'll include the fix in v2.6.1 tomorrow.

**Comment by jsvine** [comment]
Nice, and thanks!
